cfme/intelligence/chargeback/assignments.py

```diff
@@ -124,9 +128,9 @@ def step(self):
         )
 
 
-@navigator.register(Assign, 'Storage')
+@navigator.register(Assign, "Storage")
 class AssignStorage(CFMENavigateStep):
-    prerequisite = NavigateToSibling('All')
+    prerequisite = NavigateToSibling("All")
```

Comment:
Required: This navigation will not work as 'All' is a navigation step for collection
You need to use prerequisite = NavigateToAttribute('parent', 'All')